### ISSUE TITLE
Parsing error: Unexpected token = eslint because of the airbnb style configuration in React-Redux project

### ISSUE THREAD

**Issue by fabricio-garcia** [microverseinc/linters-config#87]
Since Microverse linting style is base on Airbnb this raises some times unexpected token errors due to incompatibility between the local development environment and ESLint in few specific cases.

I've been affected for this issue when trying to declare an event handler function inside a class component, the linter points to a no existent error and the only available solution that I could spot so far is to add some configuration in the .eslintrc file, like:

`"parser": "babel-eslint"`

This is the non existent error:
![Screenshot_20200714_143553](https://user-images.githubusercontent.com/9059238/87468876-f8c10680-c5df-11ea-9c61-27851198ac6b.png)

What it could be a possible solution since I'm not allowed to perform any changes in the linting configurations files and there's no other way to pass all the test?

**Comment by MauricioRobayo** [comment]
You are using the [experimental public class fields syntax](https://github.com/tc39/proposal-class-fields), which **is a proposal in stage 3**, not yet in the language.

As you notice, you are able to use it because you are using Babel which will transform it for you. That's why ESLint complains, I don't thinks is a Airbnb styleguide issue, is just that it is not in the language yet, and you have to use the `babel-eslint` parser to avoid the issue with ESLint.

The way to do it using **ES6 classes** is documented on the [React docs](https://reactjs.org/docs/handling-events.html#algolia-doc-search:~:text=When%20you%20define%20a%20component%20using,a%20method%20on%20the%20class.%20For).

You have to bind your class methods on the constructor, something like this:

```diff
export class Searchform extends Component {
+  constructor(props) {
+    super(props);
+    this.onChange = this.onChange.bind(this);
+   this.onSubmit = this.onSubmit.bind(this);
+  }

-   onChange = event => {
+  onChange(event) {
    // ...
  }

-   onChange = event => {
+  onSubmit(event) {
    // ...
  }
```

**Comment by fabricio-garcia** [comment]
That solved the issue, however other linting errors raised instead, being "destructuring assignment" the most annoying. 

It would be great if you guys remove the Airbnb style from the official linting files. Personally, I work with "eslint-recommended" along with prettier and resulting code is beautifully. 

Don't take wrong, but for me projects from Microverse are really frustrating most of the time because of the linting style implemented which causes hours of time used in fixing linting errors instead of learning something more productive.

Thank you for your suppor @MauricioRobayo!
